bundle/compliance/policy/aws_cloudtrail/data_adapter.rego

```diff
@@ -1,7 +1,7 @@
 package compliance.policy.aws_cloudtrail.data_adapter
 
-is_trail_type {
```

Comment:
I think that `not trail_items` is not needed anymore because there is a differentiation of types.